client/web/hilbert-heartbeat.js

```diff
@@ -43,7 +43,7 @@
                 debugLog("RESPONSE: " + xhttp.responseText);
             }
         };
-        var fullUrl = url + "/" + command + "?" + interval + "&appid=" + appId + "&cache_buster=" + new Date().getTime()
+        var fullUrl = url + "/" + command + "?" + (interval/1000.0) + "&appid=" + appId + "&cache_buster=" + new Date().getTime()
```

Comment:
I don't agree with dividing by 1000 in the library to convert from milliseconds to seconds. String formatting of floating point numbers can be weird and you have to be careful what you put into URLs. I'd prefer keep the interval in milliseconds (integers) and do the division on the server side.